CONTRIBUTING.md

```diff
@@ -4,7 +4,7 @@ We welcome contributions to the FireFly Project in many forms, and
 there's always plenty to do!
 
 Please visit the
-[contributors guide](https://hyperledger.github.io/firefly//contributors/contributors.html) in the
+[contributors guide](https://hyperledger.github.io/firefly/contributors/contributors.html) in the
```

Comment:
This should actually be `https://hyperledger.github.io/firefly/contributors` or `https://hyperledger.github.io/firefly/contributors/index.html`

Reply:
Ok ill make the update